celery/platforms.py

```diff
@@ -781,8 +782,54 @@ def check_privileges(accept_content):
         if not all(hasattr(os, attr)
                    for attr in ['getuid', 'getgid', 'geteuid', 'getegid']):
```

Comment:
```suggestion
                   for attr in ('getuid', 'getgid', 'geteuid', 'getegid')):
```

If we're reviewing this part of the code, we might as well improve it.